StratisCore.UI/src/app/app.component.ts

```diff
@@ -56,8 +58,18 @@ export class AppComponent implements OnInit, OnDestroy {
 
     this.subscription = stream$.subscribe(
       (data: NodeStatus) => {
-        this.loading = false;
-        this.router.navigate(['login']);
+        this.apiConnected = true;
+        this.statusIntervalSubscription = this.apiService.getNodeStatusInterval(true)
+          .subscribe(
+            response =>  {
+              const statusResponse = response.featuresData.filter(x => x.namespace === 'Stratis.Bitcoin.Base.BaseFeature');
+              if (statusResponse[0].state === 'Initialized') {
```

Comment:
Do we need to check if `statusResponse.length > 0`?

Reply:
Also, maybe `statusResponse[0].state.toLowerCase() === 'initialized'`?

Reply:
Hmm yes, I will add it to the if statement.